server/docs/game.yaml

```diff
@@ -139,3 +139,63 @@ paths:
                   value:
                     success: false
                     error: 'Player does not exist'
+
+  /game/{gameId}:
+    patch:
+      tags:
+        - 'Game'
+      summary: 'Update the dates and location of a game.'
+      description: 'This can only be done by a logged in user and a league admin.'
+      parameters:
+      - name: 'gameId'
+        in: 'path'
+        description: 'ID of the game.'
+        required: true
+        type: 'string'
+      security:
+      - BearerAuth: []
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/GameUpdate'
```

Comment:
the GameUpdate schema is actually for updating the player stats :sweat_smile: will need to make another one for updating location and date stuff, sorry bout the confusion

Reply:
okay! i'll fix it